lib/src/chart/base/base_chart/base_chart_data.dart

```diff
@@ -130,6 +131,7 @@ class SideTitles {
     ),
     this.margin = 6,
     this.interval = 1.0,
+    this.rotateAngle = 0.0,
```

Comment:
Please add it in our lerp function (see line 137) to handle the changing animation